docs/topics/multiplatform-mobile/multiplatform-mobile-integrate-in-existing-app.md

```diff
@@ -74,7 +81,7 @@ You can learn more about the [project structure](multiplatform-mobile-understand
 
 To use cross-platform code in your Android application, connect the shared module to it, move the business logic code there, and make this code cross-platform.
 
-1. Ensure that `compileSdkVersion` and `minSdkVersion` in `build.gradle.kts` of the `shared` module are the same as those in the `build.gradle` of your Android application in the `app` module.  
+1. Ensure that `compileSdk` and `minSdk` in `build.gradle.kts` of the `shared` module are the same as those in the `build.gradle` of your Android application in the `app` module.  
```

Comment:
There is a problem with staging. 
Just want to be sure, that it's OK with rendering this and the markup is not overloaded.

Reply:
I removed one of the code items and rearranged others to unload the markup